source/c.txt

```diff
@@ -75,6 +76,35 @@ Connect to MongoDB Atlas
 See `Advanced Connections <http://mongoc.org/libmongoc/current/advanced-connections.html>`__
 for more ways to connect.
 
+Versioned API
+-------------
+
+You can use the `Versioned API
+<https://docs.mongodb.com/v5.0/reference/versioned-api/>`__ feature
+starting with MongoDB Server version 5.0 and C Driver version 1.18. When
+you use the Versioned API feature, you can update your driver or server
+without worrying about backward compatibility issues for any commands
```

Comment:
Nitpick: I think you're describing "issues with commands" rather than "issues for commands" unless you're offering the commands issues for something.

Reply:
accepted